apps/src/componentLibrary/dropdown/checkboxDropdown/CheckboxDropdown.tsx

```diff
@@ -115,22 +152,7 @@ const CheckboxDropdown: React.FunctionComponent<CheckboxDropdownProps> = ({
             </li>
           ))}
         </ul>
-        <div className={moduleStyles.bottomButtonsContainer}>
-          <Button
-            type="tertiary"
-            color={buttonColors.purple}
-            text={i18n.selectAll()}
-            onClick={onSelectAll}
-            size={size}
-          />
-          <Button
-            type="tertiary"
-            color={buttonColors.purple}
-            text={i18n.clearAll()}
-            onClick={onClearAll}
-            size={size}
-          />
-        </div>
+        {getControls()}
```

Comment:
I'm not sure if we need to wrap a render function into useCallback hook, but I think we can simplify this line to smth like `{!hideControls && renderControls()}`
Or even instead of using render function, put jsx directly here and instead of destructing the props - use `rest.propName` format

Reply:
Good call, done!